docs-sources/includes/_Modules.md

```diff
@@ -3,17 +3,19 @@
 ## External helper modules
 
 Some mage modules are optional but simplify the use of the library.
+Unlike other modules, they don't expose usercommands, they just act as helpers.
```

Comment:
I think that is incorrect, e.g. archivist.

Reply:
Is there an archivist external module ?

Reply:
ah sorry, I misread that.

Reply:
I will rephrase so there is no mistake